EIPS/eip-3000.md

```diff
@@ -0,0 +1,153 @@
+---
+eip: 3000
+title: Staking Reward Calculation
+author: Tony Carson <tony.carsonn@gmail.com>, Mehmet Sabir Kiraz <m.kiraz@gmail.com>, Süleyman Kardaş <skardas@gmail.com>
+type: Standards Track
+category: ERC
+status: Draft
+created: 2020-08-28
+discussions-to: https://github.com/ethereum/EIPs/issues/3000
```

Comment:
Please keep the header in the order in the template for consistency purposes.

https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md